packages/ui/src/domains/LandingPageBuilder/LandingPageBuilder.tsx

```
@@ -1691,117 +1778,139 @@
+				</SheetTrigger>
+
+				<SheetContent>
+					<SheetHeader>
+						<SheetTitle>Configurações do Carrossel</SheetTitle>
+						<SheetDescription>
+							Edite as propriedades do carrossel.
+						</SheetDescription>
+					</SheetHeader>
+					<div className="space-y-2">
+						{items.map((item, index) => (
+							<div key={index} className="flex items-center space-x-2">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)